internal/encoding/block/from_parquet.go

```
@@ -65,13 +66,19 @@ func FromParquetBy(payload []byte, partitionBy string, filter *typeof.Schema, ap
 			columnName := cols[i]
 			columnType := schema[columnName]
 
+			fieldHandler, _ := getReverseHandler(columnType.String())
+
 			// Encode to JSON
 			if columnType == typeof.JSON {
 				if encoded, ok := convertToJSON(v); ok {
 					v = encoded
 				}
 			}
 
+			if fieldHandler != nil {
+				v, _ = fieldHandler(v)
```

We probably should not ignore the error  here